### ISSUE TITLE
Install fails with error 500 on LNbits v0.12.11 on Postgres

### ISSUE THREAD

**Issue by openoms** [riccardobl/nwcprovider#1]
Attempted install failed on my tets Raspiblitz running the latest commit on LNbits dev branch v0.12.11.
Note that it is running with PostgreSQL.

Logs when clicking install on the imported extension from this repo:
```
Sep 01 23:03:30 raspberrypi sh[2692953]: 2024-09-01 23:03:30.34 | INFO | 100.94.29.54:0 - "POST /api/v1/extension HTTP/1.1" 500
Sep 01 23:04:13 raspberrypi sh[2692953]: 2024-09-01 23:04:13.72 | INFO | 100.94.29.54:0 - "GET /api/v1/extension/nwcprovider/releases HTTP/1.1" 200
Sep 01 23:04:16 raspberrypi sh[2692953]: 2024-09-01 23:04:16.86 | INFO | 100.94.29.54:0 - "GET /api/v1/extension/release/riccardobl/nwcprovider/0.0.8 HTTP/1.1" 200
Sep 01 23:04:19 raspberrypi sh[2692953]: 2024-09-01 23:04:19.99 | INFO | lnbits.extension_manager:download_archive:483 | Downloading extension nwcprovider (0.0.8).
Sep 01 23:04:20 raspberrypi sh[2692953]: 2024-09-01 23:04:20.38 | INFO | lnbits.extension_manager:extract_archive:510 | Extracting extension nwcprovider (0.0.8).
Sep 01 23:04:20 raspberrypi sh[2692953]: 2024-09-01 23:04:20.39 | SUCCESS | lnbits.extension_manager:extract_archive:547 | Extension NWC Service Provider (0.0.8) installed.
Sep 01 23:04:20 raspberrypi sh[2692953]: 2024-09-01 23:04:20.39 | DEBUG | lnbits.core.helpers:run_migration:47 | running migration nwcprovider.1
Sep 01 23:04:20 raspberrypi sh[2692953]: running migration nwcprovider.1
Sep 01 23:04:20 raspberrypi sh[2692953]: 2024-09-01 23:04:20.40 | WARNING | lnbits.core.views.extension_api:api_install_extension:121 | (psycopg2.errors.UndefinedTable) relation "keys" does not exist
Sep 01 23:04:20 raspberrypi sh[2692953]: [SQL:
Sep 01 23:04:20 raspberrypi sh[2692953]:         CREATE TABLE nwcprovider.spent (
Sep 01 23:04:20 raspberrypi sh[2692953]:             id SERIAL PRIMARY KEY,
Sep 01 23:04:20 raspberrypi sh[2692953]:             pubkey TEXT NOT NULL,
Sep 01 23:04:20 raspberrypi sh[2692953]:             amount_msats INTEGER NOT NULL,
Sep 01 23:04:20 raspberrypi sh[2692953]:             created_at INTEGER NOT NULL,
Sep 01 23:04:20 raspberrypi sh[2692953]:             FOREIGN KEY(pubkey)  REFERENCES keys(pubkey) ON DELETE CASCADE
Sep 01 23:04:20 raspberrypi sh[2692953]:         );
Sep 01 23:04:20 raspberrypi sh[2692953]:         ]
Sep 01 23:04:20 raspberrypi sh[2692953]: (Background on this error at: http://sqlalche.me/e/13/f405)
Sep 01 23:04:20 raspberrypi sh[2692953]: 2024-09-01 23:04:20.41 | ERROR | lnbits.exceptions:http_exception_handler:93 | HTTPException 500: Failed to install extension nwcprovider (0.0.8).
Sep 01 23:04:20 raspberrypi sh[2692953]: 2024-09-01 23:04:20.41 | INFO | 100.94.29.54:0 - "POST /api/v1/extension HTTP/1.1" 500
```

Installed ok with the same method and version on a VPS where LNbits is running with SQLite.

**Comment by riccardobl** [comment]
should be fixed in the new release: [0.0.9](https://github.com/riccardobl/nwcprovider/releases/tag/0.0.9)

**Comment by openoms** [comment]
Thank you, it is working now!
